Makefile

```diff
@@ -28,13 +30,20 @@ build-ack-generate:	## Build ack-generate binary
 	@go build ${GO_TAGS} ${GO_LDFLAGS} -o bin/ack-generate cmd/ack-generate/main.go
 	@echo "ok."
 
+local-build-ack-generate:	## Build ack-generate binary using the local go.mod
```

Comment:
nit: using the file name is clearer, `Build ack-generate binary using go.local.mod`